invenio_app_ils/ui/src/invenio_app_ils/common/api/items/item.js

```diff
@@ -0,0 +1,66 @@
+import { http } from '../base';
+
+const itemURL = '/items/';
+
+const get = itemPid => {
+  return http.get(`${itemURL}${itemPid}`);
+};
+
+class ItemsQuery {
+  constructor(build) {
+    this.documentQuery = build.documentQuery ? build.documentQuery : '';
+    this.stateQuery = build.stateQuery ? build.stateQuery : '';
+  }
+
+  qs() {
+    return `${this.documentQuery}${this.stateQuery}`.replace(' AND ', '');
+  }
+
+  static get Builder() {
+    class Builder {
+      withDocPid(documentPid) {
+        this.documentQuery = documentPid
+          ? ` AND document.document_pid:${Builder.paramToQuery(documentPid)}`
```

Comment:
Bare in mind this is refactored https://github.com/inveniosoftware/invenio-app-ils/pull/147 :)

Reply:
thanks !